### DISCUSSION TITLE
Tip on making Leva wider

### DISCUSSION THREAD

**Discussion by pholmq** [pmndrs/leva#366] [Show and tell]
I finally solved how to make Leva a bit wider with CSS since it truncates text when too many nested folders is used. Also added transparency. It might be useful to others I figured:

`

      <div
        style={{
          width: 280,
          position: "absolute",
          right: 0,
          top: 0,
          zIndex: 100,
          opacity: 0.8
        }}
      >
        <Leva fill />
      </div>

`

Love this UI btw. Thank you!

**Comment by alexisdrakopoulos** [comment]
This doesn't seem to work for me on LevaPanel components :(
